src/org/omegat/core/data/TMXEntry.java

```diff
@@ -110,21 +125,12 @@ public boolean equals(Object obj) {
         if (!Objects.equals(creator, other.creator)) {
             return false;
         }
-        if (defaultTranslation != other.defaultTranslation) {
-            return false;
-        }
         if (!Objects.equals(source, other.source)) {
             return false;
         }
         return true;
     }
 
-    @Override
-    public int hashCode() {
```

Comment:
When you define `equals` you should also define `hashCode`, this is one of checkstyle warning.

Reply:
This warning is a little bit stupid: the method has been moved to concrete classes (Project,External)TMXEntry because the implementation differs, due to different internal structure.
Should we define an abstract method only to make the style checker happy?

Reply:
You can add `@SuppressWarnings` annotation here

Reply:
Patched moved to other PR, to be considered later